src/components/content/exercises/exercise-generation-wizard/topic.tsx

```diff
@@ -109,3 +109,16 @@ export const Topic: React.FC<TopicProps> = ({
     </div>
   )
 }
+
+// Helper function to extract the topic from title.
+// E.g Aufgaben zum Dreisatz => Dreisatz
+export function extractTopicFromTitle(
+  title: string | undefined
+): string | null {
+  if (!title) {
+    return null
+  }
+
+  const match = title.match(/Aufgaben zu(?:m|r)? (.+)/)
+  return match ? match[1].trim() : null
+}
```

Comment:
Just a suggestion:
Should this actually only return if the string starts with `Aufgaben zu…`?
If yes, maybe simplify the regex to something like `Aufgaben zu. (.+)`

Otherwise this should suffice:
``` ts
export function extractTopicFromTitle(title?: string){
  return title?.replace(/^Aufgaben zu./,'')
}
```